website/identifiers/utils.py

```diff
@@ -48,3 +56,80 @@ def from_anvl(data):
 
 def merge_dicts(*dicts):
     return dict(sum((each.items() for each in dicts), []))
+
+
+def get_top_level_domain(preprint):
```

Comment:
It's a bit unintuitive that this function takes a `preprint` as input. A URL would make this more intuitive and reusable.